defaultEnvironment.gradle

```diff
@@ -6,7 +6,7 @@ subprojects {
   repositories {
     mavenCentral()
     maven {
-      url "http://conjars.org/repo"
+      url "https://conjars.org/repo"
```

Comment:
LG, but any idea why are we using conjars?

Reply:
For this dependency
```
* What went wrong:
Could not resolve all files for configuration ':transportable-udfs-hive:compileClasspath'.
> Could not find org.pentaho:pentaho-aggdesigner-algorithm:5.1.5-jhyde.
  Searched in the following locations:
    - https://repo.maven.apache.org/maven2/org/pentaho/pentaho-aggdesigner-algorithm/5.1.5-jhyde/pentaho-aggdesigner-algorithm-5.1.5-jhyde.pom
    - https://repo.maven.apache.org/maven2/org/pentaho/pentaho-aggdesigner-algorithm/5.1.5-jhyde/pentaho-aggdesigner-algorithm-5.1.5-jhyde.jar
  Required by:
      project :transportable-udfs-hive > org.apache.hive:hive-exec:1.2.2 > org.apache.calcite:calcite-core:1.2.0-incubating
```